inc/abstractitiltarget.class.php

```diff
@@ -1999,7 +1999,7 @@ protected function showActorSettingsForType($actorType, array $actors) {
       );
       echo '</div>';
 
-      if ($actorType == CommonITILActor::ASSIGN) {
+      if ($actorType == CommonITILActor::ASSIGN || $actorType == CommonITILActor::OBSERVER) {
```

Comment:
Can a supplier be an observer on a ticket?

Natively, it seems that they cannot, just as they cannot be a requester. They can only be assigned.